app/reducers/index.js

```diff
@@ -89,6 +110,12 @@ export default (state = defaultState, action) => {
           },
         },
       }
+    case types.SAVE_RATING:
+      return {
+        ...saveRating(state, action.time, action.id),
```

Comment:
If your reducers cases are each held in their own function, then things work nicer here. You do not need to change anything, but ask me about this.